src/impls.rs

```diff
@@ -156,6 +156,20 @@ mod bstring {
         }
     }
 
+    impl<'a, const N: usize> From<&'a [u8; N]> for BString {
+        #[inline]
+        fn from(s: &'a [u8; N]) -> BString {
+            BString::from(&s[..])
+        }
+    }
+
+    impl<const N: usize> From<[u8; N]> for BString {
```

Comment:
`&[u8; N]` is also a type that is a pain to work with. Do you think adding an additional from impl for that type is worth it as well? Maybe on `BStr` too

Reply:
Ah BStr probably can't have one for this type for the same reason Cow can't.

Reply:
I added the `&[u8; N]` impl above this one. Did you miss it? Or am I misunderstanding you?

And nice, yeah, I actually can add a `impl<'a, const N: usize> From<&'a [u8; N]> for &'a BStr`!

Reply:
Oops there it is. Sorry I'm on my mobile and missed that one in the small screen 🙇